src/grimoirelab/core/runner/commands/run.py

```diff
@@ -83,6 +81,8 @@ def server(ctx: Context, devel: bool):
     env["UWSGI_SINGLE_INTERPRETER"] = "true"
 
     # Run maintenance tasks
+    from grimoirelab.core.scheduler.scheduler import maintain_tasks
```

Comment:
Why did you have to move this statement here?

Reply:
Because otherwise, it doesn't work. It has to initialize the Django module first, which is done here: https://github.com/chaoss/grimoirelab-core/blob/main/src/grimoirelab/core/runner/cmd.py#L58

If you try to do the import there you get `django.core.exceptions.ImproperlyConfigured: Requested setting INSTALLED_APPS, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
`